vslib/SwitchStateBaseMACsec.cpp

```diff
@@ -679,7 +681,11 @@ sai_status_t SwitchStateBase::loadMACsecAttrFromMACsecSA(
     {
         SAI_METADATA_GET_ATTR_BY_ID(attr, SAI_MACSEC_SA_ATTR_MACSEC_SSCI, attrCount, attrList);
 
```

Comment:
For easy understanding, please add a comment for describing the reason of converting ssci from host order to network order

Reply:
Sure, added it.